### ISSUE TITLE
Add resource types in relations view

### ISSUE THREAD

**Issue by recht** [komodorio/komoplane#28]
If you have compositions without setting external-name, it's very hard to see what's what without clicking through (which is pretty slow). It could also be showing the `crossplane.io/composition-resource-name` annotation value.

![Screenshot from 2023-08-17 23-39-16](https://github.com/komodorio/komoplane/assets/432100/56b6ce01-25f1-40eb-9d44-aa08f3779dd9)

**Comment by undera** [comment]
I agree, we need that improvement, already mentioned in roadmap doc. Should be not too hard to implement.

**Comment by recht** [comment]
Yes, much better :-)

**Comment by undera** [comment]
And like this, with composition names instead of resource names?

![Selection_005](https://github.com/komodorio/komoplane/assets/2026469/a53ba31d-fb40-4910-b988-633d6fdd8cf8)
